### ISSUE TITLE
[High Priority] Add More Levels of UK Output Area Boundaries

### ISSUE THREAD

**Issue by hannahblue** [CartoDB/bigmetadata#551]
We need to add these levels of UK boundaries to the DO:
![screen shot 2018-08-16 at 11 05 02 am](https://user-images.githubusercontent.com/7182102/44219865-0e56f880-a14b-11e8-8783-457d44857383.png)

Full chart for context:
[Hierarchical_Representation_of_UK_Statistical_Geographies_(Jun_2017).pdf](https://github.com/CartoDB/bigmetadata/files/2294292/Hierarchical_Representation_of_UK_Statistical_Geographies_.Jun_2017.pdf)

Also the Local Authority Districts:
http://geoportal.statistics.gov.uk/datasets/census-merged-local-authority-districts-december-2011-generalised-clipped-boundaries-in-great-britain

These will be the boundaries that feed MRLI for the UK

@antoniocarlon has already begun investigating

cc  @juanignaciosl, @stevelewis99, @alissarose, @FloBroderick

**Comment by FloBroderick** [comment]
@stevelewis99 flagged this company to me for postcode boundaries:

https://www.geoplan.com/products/map-data/uk-geoplan-open-sector-boundaries/index.html

With Geoplan we could OEM them for MRLI and our other Strategic Solutions, and then they would be maintained (vs us building and maintaining). I will find out costs tomorrow in a call and then share them with you @hannahblue. It may be prohibitive, but it will be good to find out. 

In a nutshell, we need to work out whether it makes sense to build ourselves, or buy.

**Comment by hannahblue** [comment]
This output area data has the coverage and granularity we need, and is also free and easy to access.

@stevelewis99 and I discussed and agreed today that we can pursue these boundaries as a viable alternative to postcodes for MRLI.

**Comment by hannahblue** [comment]
@stevelewis99 Flo said you have some concerns about using the Output Areas for MRLI. Can you please share them here?
